### DISCUSSION TITLE
Tempo de garantia para freela

### DISCUSSION THREAD

**Discussion by victorakio** [frontendbr/forum#1916] [Discussão]
Fala pessoal beleza?

Estou para pegar meu primeiro freela e na conversa com o cliente surgiu a dúvida de qual seria o período de garantia pelo serviço (no caso corrigir um problema que um dos plugins utilizados no site em wordpress está causando).

Como vocês costumam lidar com essa questão de garantia?

Valeu!

**Comment by fabiochiquezi** [comment]
Como quase toda resposta da área: depende uhshuhsuhushus

Bom, como é um Wordpress imagino que é um projeto simples, nesses casos eu costumo deixar aberto pro cliente que logo após entregue o projeto se encontrar algum problema ele pode me chamar, mas se o problema surgir muito tempo depois da entrega e tiver sido causado por alguém que mexeu nele, aí no caso é cobrado um custo de manutenção. (geralmente os clientes não são sem noção haha)

Mas assim, nem entra muito em detalhe, leva essa negociação numa boa nesse ponto. Sempre aproveito quando o cliente me chama dps de uns meses pra perguntar algo sobre o projeto, pra tentar fechar um novo negócio, uma nova atualização, ou peço pra ele dar estrelinha e comentar em algo a respeito haha

**Comment by victorakio** [comment]
Valeu pelas dicas @fabiochiquezi, é realmente um projeto mais simples mesmo, vou fazer isso.
